index.js

```diff
@@ -281,18 +295,22 @@ class CoverageSonar extends CoverageBase {
      *                                          - project url
      *                                          - Sonar env vars
      */
-    _getInfo({ jobId, startTime, endTime, pipelineId, prNum }) {
-        const { projectKey } = getProjectData({
+    _getInfo({ annotations, jobId, jobName, startTime, endTime, pipelineId, pipelineName, prNum }) {
+        const { projectKey, projectName } = getProjectData({
             enterpriseEnabled: sonarEnterprise,
             jobId,
-            pipelineId
+            pipelineId,
+            annotations,
+            pipelineName,
+            jobName
         });
         const infoObject = {
             envVars: {
                 SD_SONAR_AUTH_URL: sdCoverageAuthUrl,
                 SD_SONAR_HOST: sonarHost,
                 SD_SONAR_ENTERPRISE: sonarEnterprise,
-                SD_SONAR_PROJECT_KEY: projectKey
+                SD_SONAR_PROJECT_KEY: projectKey,
+                SD_SONAR_PROJECT_NAME: projectName
```

Comment:
couple of thoughts

1. can you also return SD_SONAR_PROJECT_SCOPE here ? so that it's stored as part of build model. Will help later in lookups . 
1. But is that needed ? UI or any caller with access to build, can simple call coverage with `SD_SONAR_PROJECT_KEY` ?

Reply:
Don't think it's super necessary to add since it can be inferred using SD_SONAR_PROJECT_KEY